documentation/jetty-documentation/src/main/asciidoc/operations-guide/begin/deploy.adoc

```diff
@@ -46,46 +46,46 @@ mywebapp.war
 <5> `WEB-INF/web.xml` is the web application deployment descriptor, which defines the components and the configuration of your web application.
 ====
 
-To deploy a standard web application, you need to enable the xref:og-module-deploy[`deploy` module]:
+To deploy a standard web application, you need to enable the xref:og-module-eeN-deploy[`{ee-current}-deploy` module].
+
+NOTE: The following examples assume you're deploying a Jakarta {ee-current-caps} app; for other versions of EE, make sure to activate the corresponding `{ee-all}-deploy` module.
 
 ----
-$ java -jar $JETTY_HOME/start.jar --add-module=deploy
+$ java -jar $JETTY_HOME/start.jar --add-module={ee-current}-deploy
```

Comment:
@sbordet: Based on my understanding of the updates in Jetty 12, I think this tutorial should direct users to add the EE-prefixed `eeN-deploy` module rather than the plain `deploy` module. However, please correct me if this is wrong!

Reply:
Is it correct.